micrometer-core/src/main/java/io/micrometer/core/instrument/distribution/DistributionStatisticConfig.java

```diff
@@ -148,15 +152,35 @@ public Integer getPercentilePrecision() {
         return percentilePrecision;
     }
 
+    /**
+     * @deprecated Use {@link #getMinimumExpectedValueAsDouble}. If you use this method, your code
+     * will not be compatible with code that uses Micrometer 1.4.x.
```

Comment:
1.4.x will be end of life as soon as 1.5 is released (soon™), so probably worth mentioning "1.4.x and later"